src/examples/core/fetch-asset/cli.sh

```
@@ -0,0 +1,13 @@
+# Fetch an NFT using the Metaplex CLI
+
+# Fetch asset by ID
+mplx core fetch asset <assetId>
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)